docs/src/routes/__layout-homepage.svelte

```diff
@@ -38,14 +41,10 @@
   {/if}
 </svelte:head>
 
-<SEO
- {title}
- {metadescription}
- {url}
-/>
+<SEO {title} {metadescription} {url} />
 
 <KitDocs>
-  <KitDocsLayout {navbar} --kd-content-max-width={'100%'}>
+  <KitDocsLayout {navbar} --kd-content-max-width={'100%'} search>
```

Comment:
@Adamj1232 Can we remove this too? It's all related to search, but we don't use it in this layout.

Reply:
```suggestion
  <KitDocsLayout {navbar} --kd-content-max-width={'100%'}>
```